src/Features/Core/Portable/InitializeParameter/AbstractAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -5,6 +5,7 @@
 using System;
 using System.Collections.Generic;
 using System.Collections.Immutable;
+using System.ComponentModel;
```

Comment:
why?

Reply:
Because of `typeof(InvalidEnumArgumentException).FullName!` at line 544 in this file. Yes, this exception types comes from `System.ComponentModel` namespace